<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdmChanWidget</class>
 <widget class="QFrame" name="AdmChanWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1</width>
    <height>1</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="_layoutTop">
     <item>
      <widget class="AdmIconText" name="_lblTop" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="_frameMore">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_layoutMore">
        <item>
         <widget class="AdmIconText" name="_lblMore" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AdmIconText</class>
   <extends>QFrame</extends>
   <header>src/admicontext.h</header>
  </customwidget>
  <customwidget>
   <class>AdmChanWidget</class>
   <extends>QFrame</extends>
   <header>src/admchanwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
